lib/tasks/migration.rake

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+# This file contains tasks that are used to migrate data.  It should be removed after the migration is stable
+# and proven to be without issue.
+
+namespace :pwpush do
+  desc "Run through and validate imported users."
+  task migrate_users: :environment do
+    User.find_each do |user|
+      puts "Processing user: #{user.id}/#{user.email}"
+
+      if user.name.blank?
+        # puts "  -> Setting first_name to email for user: #{user.email}"
+        user.first_name = user.email
+
+        # Saving this record also creates the default personal account
+        raise "Failed to save user: #{user.email}/#{user.errors.full_messages}" unless user.save!
+      end
+
+      puts "  -> Creating default account."
+      account = user.owned_accounts.create!(name: user.name, personal: false)
+      account.account_users.create!(user: user, admin: true)
+
+      if user.authentication_token.present?
+        puts "  -> Migrating authentication_token to API token"
+        user.api_tokens.create!(name: "Token 1", token: user.authentication_token)
```

Comment:
We probably don't need this file at all...  agree?

Reply:
And it's outdated because because safari was buggin.

Reply:
You are right, we don't need this file. So, it is removed.  I was planning to use it, then we have decided to use migration file.